lib/relay.js

```diff
@@ -192,6 +193,7 @@ function responseHandler(filterRules, config, io) {
       headers,
       requestId: headers['snyk-request-id'] || uuid(),
       streamingID,
+      maskedToken: (brokerToken.slice(0, 4) + '-...-' + brokerToken.slice(-4)),
```

Comment:
feels like the fn inputs [above](https://github.com/snyk/broker/blob/e92bd4d13398e0c00476b9ba9059d271b45e9ad8/lib/relay.js#L187) are coming from the `req`, have you checked whether you can get the `maskedToken` from there?

Reply:
As far as I can see request is [being emitted on the client](https://github.com/snyk/broker/blob/e92bd4d13398e0c00476b9ba9059d271b45e9ad8/lib/client/index.js#L77), which means request event signature would need to be updated, and hence the client... and afaik it wouldn't change much, as token isn't sent or used in every request, just upon establishing connection with broker... so it's the same if I use the `brokerToken` when initial request handler is initialized.